deploy/stacks/pipeline.py

```diff
@@ -43,6 +44,9 @@ def __init__(
         self.target_envs = target_envs
         self.repo_string = repo_string
         self.repo_connection_arn = repo_connection_arn
+        self.log_retention_duration = (
+            self.node.try_get_context('log_retention_duration') or logs.RetentionDays.TWO_YEARS.value
+        )
```

Comment:
Should we provide the enum item value as default or directly the enum key?

Reply:
Then we were doing it wrong in the mlstudio log group:

![image](https://github.com/user-attachments/assets/59c67d28-8e91-4d0e-842c-4e8a5b62945e)

Reply:
They are handled the same:
```
>>> getattr(logs.RetentionDays, 'ONE_MONTH')
<RetentionDays.ONE_MONTH: 'ONE_MONTH'>
>>> logs.RetentionDays.ONE_MONTH
<RetentionDays.ONE_MONTH: 'ONE_MONTH'>
```

Reply:
Yeap, in this case it does not really affect